app/services/user_destroyer.rb

```diff
@@ -103,7 +103,7 @@ def destroy(user, opts = {})
           StaffActionLogger.new(deleted_by).log_user_deletion(user, opts.slice(:context))
           Rails.logger.warn("User destroyed without context from: #{caller_locations(14, 1)[0]}") if opts.slice(:context).blank?
         end
-        MessageBus.publish "/logout", result.id, user_ids: [result.id]
+        MessageBus.publish "/logout/#{result.id}", result.id, user_ids: [result.id]
```

Comment:
It is necessary to include the user id 3 times here (and elsewhere)?

Reply:
The first (new addition) is the channel name.

The second argument is the message payload. I don't think we use it, so we could set it to anything. But I'd prefer not to change it, just in case there is some unknown dependence somewhere.

The third controls the permissions of the message